src/Smalot/PdfParser/Parser.php

```diff
@@ -46,16 +46,30 @@
  */
 class Parser
 {
+    /**
+     * @var Config
+     */
+    protected $config;
```

Comment:
Why not define it as `private`? (same question apply on other files)

```suggestion
    private $config;
```

Reply:
I kept the same visibility as other properties, which is `protected` and not `private`. But we can change it, especially because there is a getConfig function.